src/coffeescript.coffee

```diff
@@ -140,12 +140,14 @@ exports.compile = compile = withPrettyErrors (code, options = {}) ->
     transpiler = options.transpile.transpile
     delete options.transpile.transpile
 
+    transpilerOptions = Object.assign {}, options.transpile
```

Comment:
Yeah, indeed. :) I'm avoiding using them in the codebase, though, because the compiler is supposed to stay compatible with Node 6 and at some point that syntax will reach Stage 4 and we'll start outputting it as is.

Reply:
Any other notes?

Reply:
Nope, LGTM.